codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -196,4 +359,13 @@ protected ZlibWrapper determineWrapper(String acceptEncoding) {
         }
         return null;
     }
+
+    /**
+     * Check if {@link Brotli} is available for compression or not.
+     *
+     * @return Returns {@code true} if {@link Brotli} is available else {@code false}
+     */
+    public static boolean isAvailable() {
```

Comment:
Could this be moved somewhere else? We could add a new utility class such as `Brotli`, so that a user can call `Brotli.isAvailable()`?

Reply:
Removed